ncp.sh

```diff
@@ -22,8 +23,8 @@ install()
 {
   # NCP-CONFIG
   apt-get update
-  $APTINSTALL git dialog whiptail jq
-  mkdir -p "$CONFDIR" "$BINDIR"
+  $APTINSTALL git dialog whiptail jq tmux locales-all
```

Comment:
ditto

Reply:
Implemented in e2edadf7b0c20d59b8b681ec5aa29b4462fb9e10